sdk/servicebus/service-bus/test/utils/testutils2.ts

```diff
@@ -265,7 +274,7 @@ export class ServiceBusTestHelpers {
    * The receiver created by this method will be cleaned up by `afterEach()`
    */
   async getPeekLockReceiver(
-    entityNames: ReturnType<typeof getEntityNames>
+    entityNames: Omit<ReturnType<typeof getEntityNames>, "isPartitioned">
```

Comment:
Why do we need to exclude isPartitioned?

Reply:
"isPartitioned" is a required parameter, we shouldn't need it to get a receiver, hence removing it.

Reply:
Also makes sense :)